python/lsst/ip/diffim/metrics.py

```diff
@@ -68,12 +68,12 @@ class NumberSciSourcesMetricTask(MetricTask):
     ConfigClass = NumberSciSourcesMetricConfig
 
     def run(self, sources):
-        """Count the number of non-primary science sources.
+        """Count the number of primary science sources.
 
         Parameters
         ----------
-        sources : `lsst.afw.table.SourceCatalog` or `None`
-            A science source catalog, which may be empty or `None`.
+        sources : `lsst.afw.table.SourceCatalog`.
```

Comment:
```suggestion
        sources : `lsst.afw.table.SourceCatalog`
```
I don't think we usually put punctuation here.